R/msgmerge.R

```diff
@@ -70,3 +79,66 @@ update_en_quot_mo_files <- function(dir, verbose) {
   }
   return(invisible())
 }
+
+
+
+#' Create or update a `.po` file containing translations
+#'
+#' @description
+#' * `po_create()` creates a new `po/{languages}.po` containing the messages to be
+#'   translated.
+#' * `po_update()` updates an existing `.po` file after the messages in a
+#'   package have changed. The translations for existing messages are preserved;
+#'   new messages are added; and translations for deleted message are marked
+#'   as deprecated and moved to the bottom of the file.
+#'
+#' @param languages Language identifiers. These are typically two letters (e.g.
```

Comment:
Wording here implies it's vectorized but we use `file.exists(po_path)` below... which do we want to fix? I would guess vectorization is better to support

Reply:
I've played around with this a bit, and I think it'll be easiest to implement simultaneously with adding support for src translations (oops!) after #234 provides `po_files()` (or whatever it gets called).